src/runtime/server/services/authjs/nuxtAuthHandler.ts

```diff
@@ -1,69 +1,70 @@
 
-const useConfig = () => useTypedBackendConfig(useRuntimeConfig(), 'authjs')
+// const SUPPORTED_ACTIONS: AuthAction[] = ['providers', 'session', 'csrf', 'signin', 'signout', 'callback', 'verify-request', 'error', '_log']
 
 /**
  * Parse a body if the request method is supported, return `undefined` otherwise.
 
 * @param event H3Event event to read body of
  */
-const readBodyForNext = async (event: H3Event) => {
-  let body: any
+// const readBodyForNext = async (event: H3Event) => {
+//   let body: any
 
-  if (isMethod(event, ['PATCH', 'POST', 'PUT', 'DELETE'])) {
-    body = await readBody(event)
-  }
-  return body
-}
+//   if (isMethod(event, ['PATCH', 'POST', 'PUT', 'DELETE'])) {
+//     body = await readBody(event)
+//   }
+//   return body
+// }
 
 /**
  * Get action and optional provider from a request.
  *
  * E.g., with a request like `/api/signin/github` get the action `signin` with the provider `github`
  */
-const parseActionAndProvider = ({ context }: H3Event): { action: AuthAction, providerId: string | undefined } => {
-  const params: string[] | undefined = context.params?._?.split('/')
+// const parseActionAndProvider = ({ context }: H3Event): { action: AuthAction, providerId: string | undefined } => {
+//   const params: string[] | undefined = context.params?._?.split('/')
 
-  if (!params || ![1, 2].includes(params.length)) {
-    throw createError({ statusCode: 400, statusMessage: `Invalid path used for auth-endpoint. Supply either one path parameter (e.g., \`/api/auth/session\`) or two (e.g., \`/api/auth/signin/github\` after the base path (in previous examples base path was: \`/api/auth/\`. Received \`${params}\`` })
-  }
+//   if (!params || ![1, 2].includes(params.length)) {
+//     throw createError({ statusCode: 400, statusMessage: `Invalid path used for auth-endpoint. Supply either one path parameter (e.g., \`/api/auth/session\`) or two (e.g., \`/api/auth/signin/github\` after the base path (in previous examples base path was: \`/api/auth/\`. Received \`${params}\`` })
+//   }
 
-  const [unvalidatedAction, providerId] = params
+//   const [unvalidatedAction, providerId] = params
 
-  // Get TS to correctly infer the type of `unvalidatedAction`
-  const action = SUPPORTED_ACTIONS.find(action => action === unvalidatedAction)
-  if (!action) {
-    throw createError({ statusCode: 400, statusMessage: `Called endpoint with unsupported action ${unvalidatedAction}. Only the following actions are supported: ${SUPPORTED_ACTIONS.join(', ')}` })
-  }
+//   // Get TS to correctly infer the type of `unvalidatedAction`
+//   const action = SUPPORTED_ACTIONS.find(action => action === unvalidatedAction)
+//   if (!action) {
+//     throw createError({ statusCode: 400, statusMessage: `Called endpoint with unsupported action ${unvalidatedAction}. Only the following actions are supported: ${SUPPORTED_ACTIONS.join(', ')}` })
+//   }
 
-  return { action, providerId }
-}
+//   return { action, providerId }
+// }
```

Comment:
do you want to get rid of all this commented out code, so that we can do a cleaner review?

Reply:
Absolutely yes, it's here because the PR is not fully polished.
I will remove this after a bit more testing